### ISSUE TITLE
Cannot find publish() function

### ISSUE THREAD

**Issue by kencoken** [rever-ai/CombineFirebase#21]
Apologies if am missing something as a bit of a Combine novice. I have installed CombineFirebase 0.3.0 and Firebase 7.11.1 and am trying to follow the example from the readme file for listening to a collection in Firestore:

```swift
import CombineFirebase
import Firebase
import Combine

let db = Firestore.firestore()
var cancelBag = Set<Cancellable>()

func listenCollection() {
    db.collection("cities")
        .publisher()
        .sink(receiveCompletion: { completion in
            switch completion {
            case .finished: print("🏁 finished")
            case .failure(let error): print("❗️ failure: \(error)")
            }
        }) { snapshot in
            print("collection data: \(snapshot.documents)")
        }.store(in: &cancelBag)
}
```

However, Swift is unable to find the `publisher()` method on `db.collection("cities")` with only a:
```swift
publisher(for keyPath: KeyPath<CollectionReference, Value>)
```

I can see this exists on the Query object on the source code [here](https://github.com/rever-ai/CombineFirebase/blob/master/Sources/Firestore/Query%2BCombine.swift#L34) so am unsure why `db.collection("cities")` which returns a `CollectionReference` ([which derives from Query](https://firebase.google.com/docs/reference/swift/firebasefirestore/api/reference/Classes/CollectionReference)) can not find this definition.

**Comment by kencoken** [comment]
I can further confirm that this only happens when I install the package using the Swift Package Manager and not CocoaPods.

If importing the relevant sub-package directly then the compilation works:
```swift
import CombineFirebaseFirestore
```

So it seems that something is going wrong with the global import.

**Comment by kshivang** [comment]
Yes you are right, import issue is there with SPM version, you have to use respective module import as of now, new PR is welcomed on this.
